{"hierarchy":{"paths":[["doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore","doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"]]},"metadata":{"roleHeading":"Case","modules":[{"name":"DigiMeCore"}],"symbolKind":"case","title":"SDKError.healthDataErrorError(error:)","externalID":"s:10DigiMeCore8SDKErrorO015healthDataErrorG0yACs0G0_p_tcACmF","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"healthDataErrorError"},{"kind":"text","text":"("},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}]},"schemaVersion":{"patch":0,"major":0,"minor":3},"sections":[],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"healthDataErrorError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}],"kind":"declarations"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/digimecore\/sdkerror\/healthdataerrorerror(error:)"]}],"abstract":[{"type":"text","text":"Authorization failed with the specified error"}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/healthDataErrorError(error:)"},"kind":"symbol","references":{"doc://digimecore.DigiMeCore/documentation/DigiMeCore":{"url":"\/documentation\/digimecore","abstract":[],"role":"collection","identifier":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore","kind":"symbol","title":"DigiMeCore","type":"topic"},"doc://digimecore.DigiMeCore/documentation/DigiMeCore/SDKError":{"kind":"symbol","role":"symbol","navigatorTitle":[{"text":"SDKError","kind":"identifier"}],"type":"topic","url":"\/documentation\/digimecore\/sdkerror","abstract":[{"text":"SDK Errors","type":"text"}],"title":"SDKError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"SDKError"}],"identifier":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"},"doc://digimecore.DigiMeCore/documentation/DigiMeCore/SDKError/healthDataErrorError(error:)":{"abstract":[{"text":"Authorization failed with the specified error","type":"text"}],"role":"symbol","title":"SDKError.healthDataErrorError(error:)","kind":"symbol","type":"topic","identifier":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/healthDataErrorError(error:)","url":"\/documentation\/digimecore\/sdkerror\/healthdataerrorerror(error:)","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"healthDataErrorError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}]}}}